src/games/strategy/triplea/ui/MapRouteDrawer.java

```diff
@@ -63,6 +65,7 @@ public void drawRoute(final Graphics2D graphics, final RouteDescription routeDes
     final int imageWidth = mapPanel.getImageWidth();
     final int imageHeight = mapPanel.getImageHeight();
     final Dimension imageDimension = new Dimension(imageWidth, imageHeight);
+    screenWidth = mapPanel.getWidth();
```

Comment:
So 1. at minimum, pass in the value as a param instead of a static state
2. The method in question looks like it should live with MapPanel.